src/Persistent/Pure.hs

```diff
@@ -245,6 +246,9 @@ runPersistExcept :: PersistExcept -> ServantErr
 runPersistExcept (PersistError500 msg)            = err500 { errBody = cs msg }
 runPersistExcept (PersistError404 msg)            = err404 { errBody = cs msg }
 runPersistExcept (PersistErrorNotImplemented msg) = err500 { errBody = cs msg }
+runPersistExcept (UserLoginInUse li) =
+    err500 { errBody = "user login in use: " <> cs (show li) }
+    -- FIXME: what's a good status code for 'login in use'?
```

Comment:
On Tue, Apr 12, 2016 at 06:02:27PM -0700, Mikolaj Konarski wrote:
> Date: Tue, 12 Apr 2016 18:02:27 -0700
> From: Mikolaj Konarski <notifications@github.com>
> To: liqd/aula <aula@noreply.github.com>
> Cc: fisx <mf@zerobuzz.net>
> Subject: Re: [liqd/aula] Misc fixes. (#358)
>
> > @@ -245,6 +246,9 @@ runPersistExcept :: PersistExcept -> ServantErr
> >  runPersistExcept (PersistError500 msg)            = err500 { errBody = cs msg }
> >  runPersistExcept (PersistError404 msg)            = err404 { errBody = cs msg }
> >  runPersistExcept (PersistErrorNotImplemented msg) = err500 { errBody = cs msg }
> > +runPersistExcept (UserLoginInUse li) =
> > +    err500 { errBody = "user login in use: " <> cs (show li) }
> > +    -- FIXME: what's a good status code for 'login in use'?
>
> 200? USers are know to reload pages and go back, etc. Otherwise, perhaps 403:  https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.4.4

we should make sure that page reload on forms does not happen, and i think we do (see #83).  so 403 is probably the right one.  will fix, thanks!